packages/expo-cli/src/commands/start/TerminalUI.js

```diff
@@ -107,9 +118,104 @@ export const startAsync = async (projectDir, options) => {
 
   startWaitingForCommand();
 
-  await printServerInfo(projectDir);
+  await printServerInfo(projectDir, options);
 
   async function handleKeypress(key) {
+    if (options.webOnly) {
+      switch (key) {
+        case 'a':
+          clearConsole();
+          log('Trying to open the web project in Chrome on Android...');
+          await Android.openWebProjectAsync(projectDir);
+          printHelp();
+          break;
+        case 'i':
+          clearConsole();
+          log('Trying to open the web project in Safari on the iOS simulator...');
+          await Simulator.openWebProjectAsync(projectDir);
+          printHelp();
+          break;
+        case 'e':
+          log(chalk.red` \u203A Sending a URL is not supported in web-only mode`);
```

Comment:
Why not? Could be nice for testing on physical devices.

Reply:
figured we could add it later